docs/resources/gaussdb_redis_instance.md

```diff
@@ -135,13 +145,18 @@ The `backup_strategy` block supports:
   0 to 35. If this parameter is set to 0, the automated backup policy is not set. If this parameter is not transferred,
   the automated backup policy is enabled by default. Backup files are stored for seven days by default.
 
+The `public_ips` block supports:
+
+* `id` - (Required, String) Specifies ID of the public ip.
+
+* `node_id` - (Optional, String) Specifies the ID of the GaussDB redis node.
```

Comment:
Is that possible that you already knew the node id when creating the instance? As my understanding, after the instance created the nodes will be known.